store/moduleState.js

```diff
@@ -1,9 +1,12 @@
-const availableStates = ['idle', 'loading', 'error', 'success']
+const availableStates = ['idle', 'loading', 'error', 'success', 'error.storeCode', 'error.403', 'error.noLenght', 'error.storeEmail', 'error.missingConfiguredProcessId']
```

Comment:
```suggestion
const availableStates = ['idle', 'loading', 'error', 'success', 'error.storeCode', 'error.403', 'error.noLength', 'error.storeEmail', 'error.missingConfiguredProcessId']
```

és akkor nyílván használatakor is